File: Java/Braking/src/test/java/tddmicroexercises/braking/BrakeTest.java
package tddmicroexercises.braking;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class BrakeTest {

    @Test
    public void testPowerDissipation() throws Exception {
        Brake brake = new Brake();
        assertEquals(330000.0, brake.powerDissipation(110), 0.1);
    }
}

File: scala/src/test/scala/tddmicroexercises/braking/BrakeTest.scala
package tddmicroexercises.braking

import org.scalatest.FlatSpec
import org.scalatest.Matchers

class BrakeTest extends FlatSpec with Matchers {

  it should  "testPowerDissipation" in {
    val brake: Brake = new Brake
    brake.powerDissipation(110) should be (330000.0 +- 0.1)
  }
}